File: BackEnd/QuezalliApi/QuezalliApi/Models/RecetaPulsera.cs
using System.ComponentModel.DataAnnotations;

namespace QuezalliApi.Models
{
    public class RecetaPulsera
    {

        [Key]
        public int IdRecetaPulsera { get; set; }

        
        public string Nombre { get; set; } = null!;

        
        public string Descripcion { get; set; } = null!;

        
        public string Foto { get; set; } = null!;

        public float CostoProduccion { get; set; }

        public float PrecioVenta { get; set; }

        public string Observaciones { get; set; } = null!;



        [Required]
        public string Estatus { get; set; } = null!;



    }
}

File: FrontEnd/ProyectoQuetzalli-Angular/src/app/Interfaces/recetaPulsera.ts
export interface Material {
  idPulseraMaterial: number;
  idRecetaPulsera: number;
  idMaterial: number;
  cantidad: number;
  // Otros campos relacionados con los materiales
}

export interface RecetaPulsera {
  idRecetaPulsera: number;
  nombre: string;
  descripcion: string;
  foto: string;
  costoProduccion: number;
  precioVenta: number;
  observaciones: string;
  estatus: string;
  materiales?: Material[];
}
